src/interfaces/nameRequest-interface.ts

```diff
@@ -0,0 +1,39 @@
+import { CorpTypeCd } from '@/enums'
+
+/**
+ * Name Request State interface.
+ * (Used by our UI/API - not the same as namex response object.)
+ * See:
+ * https://github.com/bcgov/business-schemas/blob/master/src/registry_schemas/schemas/name_request.json
+ */
+/** Name request response details interface. */
+export interface NameRequestDetailsIF {
+  approvedName: string
+  status: string
+  consentFlag: string
+  expirationDate: string
+}
+
+/** Name request applicant details interface. */
+export interface NameRequestApplicantIF {
+  firstName: string
+  middleName: string
+  lastName: string
+  emailAddress: string
+  phoneNumber: string
+  addressLine1: string
+  addressLine2: string
+  addressLine3: string
+  city: string
+  countryTypeCode: string
+  postalCode: string
+  stateProvinceCode: string
+}
+
+export interface NameRequestIF {
```

Comment:
The bcrs-shared-components compiler insisted on having this last (or rather, what it depends on above it).